extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -118,13 +121,81 @@
   transform-origin: right !important;
 }
 
+[info] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: 48px !important; 
+}
+
+[info] [dir=rtl] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: auto !important; 
+  padding-left: 48px !important; 
+}
+
+.i-amphtml-message-container {
+  width: max-content !important;
+  transition-property: opacity, transform !important;
+  float: right !important;
+  position: absolute !important;
+  top: 14px !important;
+  right: 48px !important;
+  font-family: 'Roboto', sans-serif !important;
+}
+
+[dir=rtl] .i-amphtml-message-container {
+  right: auto !important;
+  left: 48px !important;
+}
+
+[messagedisplay="noshow"] .i-amphtml-message-container {
+  transition: opacity 0.2s cubic-bezier(0.33, 0.33, 1, 1), transform 0.2s cubic-bezier(0.1, 0.4, 1, 1) 0.5s !important;
```

Comment:
This is the class that's applied to hide the message, so these curves will control the "exit" animation.

Material design treats animations within the screen differently than it treats elements that move in and out of screen bounds.
For animations within the screen bound, they recommend using the "standard curve" (ease in out): your `transform` statements should use `cubic-bezier(0.4, 0.0, 0.2, 1)`.

Because they're within the screen we should keep them fast, and `0.2s` seems reasonable.

About the opacity, goal would be to spend most of the animation time on the higher opacity percents. That means the animation should accelerate when setting the opacity to 0, and decelerate when setting the opacity to 1.

Curve to set opacity to 0 should be: `cubic-bezier(0.4, 0.0, 1, 1)`
Curve to set the opacity to 1 should be: `cubic-bezier(0.0, 0.0, 0.2, 1)`

Let me know if you have any question about this or if you need pointers to the internal doc :))